content/docs/user-guide/editor/viewport/_index.md

```diff
@@ -1,40 +1,121 @@
 
-This feature has the following limitations:
+![Image of reference space widget](/images/user-guide/editor/reference-space-widget.png)
 
-+ Custom shortcuts for the Viewport Interaction Model are not supported.
+| Mode | Keyboard Hotkey | Description |
+| - | - | - |
+| World space | Hold **Shift** | Manipulators align with the unchanging world space that is common to all entities in a level. |
+| Parent space | | If an entity has a parent, manipulators align with the parent's rotation. |
+| Local space | | Manipulators align with the rotation of the entity that is selected. |
 
-**Topics**
+### Component mode switcher
 
-+ [Modifying the Transform](/docs/user-guide/editor/viewport/viewport-interaction-model-transform/)
-+ [Reference Space Model](/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model/)
-+ [Using Group Selections and Reference Spaces](/docs/user-guide/editor/viewport/group-selections-reference-spaces/)
-+ [Resetting the Transform](/docs/user-guide/editor/viewport/reset-transform/)
-+ [Matching the Transform](/docs/user-guide/editor/viewport/transform-matching/)
-+ [Using Pivot Mode](/docs/user-guide/editor/viewport/pivot-mode/)
-+ [Using the Camera Space](/docs/user-guide/editor/camera-space/)
+![Image of component mode switcher widget](/images/user-guide/editor/component-mode-switcher.png)
+
+Every entity in the 3D viewport has a Transform component.  Other components that you can add to entities have properties that are easily edited in the 3D viewport using the component's _edit mode_.  You can **left-click** the icon of an editable component to enter that component's edit mode.  The Transform component's edit mode is selected by default. 
+
+The following table lists some of the components with an edit mode and describes the tasks you can complete while it is selected in the component mode switcher.  Refer to a component's page in the [Component Reference](/docs/user-guide/components/reference/) for a description of actions you can take while in edit mode.
+
+| Component | Edit Mode Tasks |
+| - | - |
+| [Shape](\docs\user-guide\components\reference\shape) | Edit the dimensions of the various shape components. |
+| [Whitebox](\docs\user-guide\components\reference\shape\white-box) | Design models with the Whitebox tool. |
```

Comment:
update: `Whitebox` -> `White Box Tool` (we always separate the words and capitalize)